src/assets/browser/app/src/composit/stage/stage.rs

```diff
@@ -1,12 +1,13 @@
 use std::collections::HashMap;
```

Comment:
(Whole file) separate "stage" position management into own module. Only lightly coupled with rest of package and easier to factor when methods are private to a small module.